### DISCUSSION TITLE
How to install the Home Assistant Add-on

### DISCUSSION THREAD

**Discussion by baobab67** [allangood/rtlamr2mqtt#72] [Q&A] [answered]

**Comment by allangood** [comment, marked as answer]
Hi @baobab67 ,

This is something that I need help actually. As I don't use or have Home Assistant supervised installed, I don't have a good way to test it. The whole addon work was made by @AnthonyPluth 

I have to setup a lot of things first to have a good documentation on how this work.

I am going to setup everything to have it properly documented.

**Reply by AnthonyPluth** [reply]
@baobab67 - Instructions below:

- Navigate to your Add-Ons (Configuration > Add-ons, Backups, & Supervisor)
- Click the Add-On Store button
- Navigate to Repositories (3 dots in the top-right corner > Repositories)
- Add this repository (https://github.com/allangood/rtlamr2mqtt) and click 'Add'
- You should now see the 'rtlamr' Add-On at the bottom of your Add-On Store. Click to install and configure.

**Reply by eCyclops** [reply]
Thanks for the instructions. In current HAOS, it's required to enable "Advanced Mode" in your User Profile before the added repository appears.
I've installed, and I can see the rtlamr in MQTT Explorer (running on another machine), but I don't see the output from my meter. I got the id off the front of the meter, so that should be okay.

**Reply by eCyclops** [reply]
And, I realized that I had assumed I was getting scm+. It turns out my meter does scm, so fixing that fixed it.
